_posts/2023-02-01-solidity-0.8.18-release-announcement.md

```diff
@@ -87,3 +87,30 @@ We highly recommend reviewing the detailed list of breaking changes in our docum
 We advise all Solidity developers to upgrade to version 0.8.18 to take advantage of these improvements and optimizations. Our team remains dedicated to continuously enhancing Solidity to benefit the entire Ethereum ecosystem.
 
 Get ready to unleash your creativity and build impactful decentralized applications with the latest version of the Solidity Compiler. Happy coding!
+
+## New linux binaries released for Solidity 0.8.18
+
+The Linux binary for the 0.8.18 release was originally built against a too recent version of glibc and thereby did not run on older Linux systems. 
+Therefore, we rebuilt and replaced the binary so that you may see two different hashes for the ``solc-linux-amd64-v0.8.18+commit.87f61d96``. 
+Both versions of the compiler will behave identically other than the fact that only the new one will work on systems with glibc versions older than 2.32.
```

Comment:
```suggestion
Both versions of the compiler will behave identically other than the fact that only the new one will work on systems with glibc versions older than 2.32.

To be clear: both binaries will produce identical outputs under all circumstances, including the commit hash in the metadata. Only the hash of the compiler binary itself will change due to the replacement, but the new binary will always produce byte-identical output.
```

Since it came up in solc-tooling, maybe better to be crystal clear on this part?